app/src/main/res/values/strings.xml

```diff
@@ -63,111 +63,111 @@
     <string name="txt3_tutorial_listen">If you don\'t like the clip, you can tap on this button to skip and get another one.</string>
-    <string name="txt4_tutorial_listen">Tap on this icon to play the clip to validate.</string>
-    <string name="txt5_tutorial_listen">Tap on this icon to stop the listening of the clip.</string>
-    <string name="txt6_tutorial_listen">Tap on this icon if you want to listen again the clip.</string>
-    <string name="txt7_tutorial_listen">Tap on this icon if the clip coincides with the sentence, so to accept it.</string>
-    <string name="txt8_tutorial_listen">Tap on this icon if the clip doesn\'t coincide with the sentence, so to reject it.</string>
+    <string name="txt4_tutorial_listen">Tap on this icon to play the clip.</string>
+    <string name="txt5_tutorial_listen">Tap on this icon to stop playing the clip.</string>
+    <string name="txt6_tutorial_listen">Tap on this icon if you want to listen to the clip again.</string>
+    <string name="txt7_tutorial_listen">If the clip matches the sentence, tap on this icon to accept it.</string>
+    <string name="txt8_tutorial_listen">If the clip doesn\'t match the sentence, tap on this icon to reject it.</string>
     <!-- NotAvailableNow -->
-    <string name="txtNotAvailableNow">So sorry, this feature isn\'t available now.\nWe\'re working constantly to improve the app.</string>
+    <string name="txtNotAvailableNow">Sorry, this feature doesn\'t exist yet.\nWe\'re working constantly to improve the app, so check back later to see if it\'s been added.</string>
     <!-- Badges -->
     <string name="btnCloseBadges">Close</string>
     <string name="labelAllBadges">All badges</string>
-    <string name="new_badge_earnt_message">Hooray! You have just earnt a new badge.</string>
+    <string name="new_badge_earnt_message">Hooray! You have just earned a new badge.</string>
     <!-- Message dialog -->
     <string name="btnOk">Ok</string>
     <string name="messageDialogErrorTitle">Error</string>
-    <string name="messageDialogGenericError">I\'m so sorry. Something didn\'t work.\nPlease, try again or contact the developer.</string>
-    <string name="messageDialogErrorCode">Something didn\'t work. Please, try again or contact the developer.\nError code: EX-{{*{{error_code}}*}}</string>
+    <string name="messageDialogGenericError">That didn\'t work.\nPlease try again, or contact the developer.</string>
+    <string name="messageDialogErrorCode">That didn\'t work. Please try again, or contact the developer.\nError code: EX-{{*{{error_code}}*}}</string>
     <string name="youHaveToAcceptPrivacyPolicyTitle">Login failed</string>
-    <string name="youHaveToAcceptPrivacyPolicy">You need to do the login on Common Voice website and you have to accept Privacy Policy.</string>
+    <string name="youHaveToAcceptPrivacyPolicy">You must first log in on the Common Voice website and accept the Common Voice privacy policy.</string>
```

Comment:
Should "log in" is a verb and "login"/"log-in" the noun? https://dictionary.cambridge.org/dictionary/english/log-in

I think here you should say "log-in", right?

Reply:
That's really interesting. In all my time alive, I have never seen "log-in" written with a hyphen. I guess it makes sense that log-in is a noun.

However, while the original sentence was a noun, I actually changed the grammar of the sentence to make "log in" a verb, so according to your dictionary link, I am correct.

Reply:
@cloudrac3r no problem. It was just a curiosity 😄 you are a native, of course you can speak better than me English!